frontend/src/container/PipelinePage/PipelineListsView/Preview/hooks/usePipelinePreview.ts

```diff
@@ -53,7 +53,7 @@ const usePipelinePreview = ({
 		isLoading: isFetching,
 		outputLogs,
 		isError,
-		errorMsg: error?.response?.data?.error || '',
+		errorMsg: error?.message || '',
```

Comment:
Rather than passing an empty string, we have a `SOMETHING_WENT_WRONG` constant that can be used here.

Reply:
We need to rewrite the Request / Response handling flow. We can take it up later.